### PR TITLE
Force preview and thumbnails to https

### PR DESCRIPTION
**Pull request by curioussavage**
httpsProxy bool should have been checked too

:eyeglasses: @ajacksified

### CODE REVIEW

File: src/views/components/ListingContent.jsx [hunk 1]
```diff
@@ -439,7 +442,8 @@ class ListingContent extends BaseComponent {
   }
 
   _buildThumbnail(listing, expand, urlIsImage, isNSFW, playable, preview) {
-    let https = this.props.app.config.https;
+    const config = this.props.app.config
+    let https = config.https || config.httpsProxy;
```
Comment (ajacksified) [inline src/views/components/ListingContent.jsx]: `const`

**Comment by ajacksified** [comment]
:nail_care:

**Comment by curioussavage** [comment]
:haircut:

**Comment by ajacksified** [comment]
:fish:

**Pull request closed without merging**